generics/constants/api-responses.js

```diff
@@ -25,20 +25,38 @@ module.exports = {
     ORG_DEACTIVATED: "Organisation Deactivated Successfully",
     ORG_UPDATED: "Organisation Details Updated Successfully",
     FAILED_TO_ADD_USER_TO_ORG: "Failed to add user to organisation",
+    ROLES_REMOVED: "Roles removed successfully",
```

Comment:
@rakeshSgr - Remove comments and extra new lines

Reply:
resolved